src/HasTranslations.php

```diff
@@ -63,6 +67,26 @@ public function translationExists(string $locale): bool
         return $this->translations()->where($this->getLocaleKeyName(), $locale)->exists();
     }
 
+    /**
+     * Get all the translations.
+     *
+     * @return \Illuminate\Database\Eloquent\Collection
+     */
+    public function getTranslations()
+    {
+        return $this->translations()->get();
```

Comment:
Wouldn't this be the same as `return $this->translations;`?

Reply:
improves the type hinting and doesn't rely on getAttribute, this also prevents that somebody can make an accessor that ruins the saving logic (as we directly get the relationship value)